ironfish/src/wallet/exporter/accountImport.ts

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Account } from '../account/account'
+import { MultisigKeysImport } from '../interfaces/multisigKeys'
+import { HeadValue } from '../walletdb/headValue'
+
+export type AccountImport = {
```

Comment:
Might consider adding a comment with some guidance for how this type should be used, for example if `AccountImport` should only be used as an input/output for the encoders.

Reply:
I'm planning to do that in a file `wallet/exporter/README.md` in another PR but I can also add that here too. Are you  OK if that's in another PR? I'm going to rename this type in another PR next to `AccountExported`